web-next/app/code-review/sonar-qube/page.module.css

```
@@ -1,1094 +1,684 @@
+  color: var(--accent-strong);
+  padding: 0.15em 0.4em;
+  border-radius: 5px;
   font-family: var(--font-mono);
-  font-size: 0.8rem;
-  font-weight: 700;
-  color: var(--layer-color, var(--neon-green));
-  min-width: 150px;
-  flex-shrink: 0;
+  font-size: 0.88em;
+  overflow-wrap: break-word;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)